asset_dashboard/static/js/components/map_utils/zoomToSearchGeometries.js

```diff
@@ -0,0 +1,17 @@
+export default function zoomToSearchGeometries(map, group) {
+    map.eachLayer((layer) => {
+      if (layer.feature?.properties.identifier) {
+        // If "identifier" is in "properties",
+        // then this is a feature from the search.
+        // This is extremely dependent on the search geometries
+        // having this "identifier" key, so beware!
+        // I haven't been able to figure out a better
+        // way of zooming in on the search results.
```

Comment:
Hmmmm, is there a more explicit way to identify search results? Like, adding a method attribute like "source: search" or something to features returned from the API? That feels hacky, too... what do you think?

Reply:
a method attribute in the geojson response? would this be added to the serializer?

Reply:
here's a sample of geojson from the search:

```
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "geometry": {
        "type": "LineString",
        "coordinates": [
          [-87.87729052716881, 41.70864290520196],
          [-87.87722544938457, 41.708603704430736],
          [-87.87716885402861, 41.708534150345045],
          [-87.87713091829738, 41.70845537603724]
        ]
      },
      "properties": {
        "identifier": 3273,
        "name": "Little Red Schoolhouse Nature Center Trails"
      }
    },
  ]
}
```

are are you suggesting we put it in the properties or in a higher level, like
```
"type": "FeatureCollection",
"source": "search",
  "features":
  ...
```

i think that would work, though not entirely how with drf. but i'm sure we could figure it out.

Reply:
> a method attribute in the properties of the geojson response? would this be added to the serializer?

Yes, exactly, so it would show up as attribute in the properties of each feature. It's just one idea, open to others.

Reply:
yeah i think that's better than what i'm doing here because it's more explicit without that big code comment.

Reply:
@hancush like so? 

i made a [source serializer class](https://github.com/fpdcc/ccfp-asset-dashboard/blob/99dfc421b4e0f98e90fd86397aa7c88e83c706e6/asset_dashboard/serializers.py#L101) for [overriding when we need that field](https://github.com/fpdcc/ccfp-asset-dashboard/blob/99dfc421b4e0f98e90fd86397aa7c88e83c706e6/asset_dashboard/serializers.py#L110). and that helped [clean up the javascript](https://github.com/fpdcc/ccfp-asset-dashboard/blob/99dfc421b4e0f98e90fd86397aa7c88e83c706e6/asset_dashboard/static/js/components/map_utils/zoomToSearchGeometries.js#L3). (sorry these pieces point at links and not one commit, i committed and then realized i need to reuse the code, so refactored)